bridge/contracts/nftbridge/NFTBridge.sol

```diff
@@ -148,35 +148,33 @@ contract NFTBridge is
       "Bridge: Unknown token"
     );
     require(_to != NULL_ADDRESS, "Bridge: Null To");
-    require(_amount > 0, "Bridge: Amount 0");
     require(_blockHash != NULL_HASH, "Bridge: Null BlockHash");
     require(_transactionHash != NULL_HASH, "Bridge: Null TxHash");
```

Comment:
@pmprete should we require a non-null from address? I left it like this since we didn't check it in the original bridge, but I'm not sure about the reason

Reply:
From bridge is not as sensitive as the to, but i think it's a good idea to check that value

Reply:
awesome, will do